src/LandIce/problems/LandIce_ParamEnum.hpp

```diff
@@ -19,7 +19,9 @@ enum class ParamEnum
   Mu,
   Power,
   Homotopy,
-  GLHomotopy
+  GLHomotopy,
+  Theta_0,
```

Comment:
You already have Theta_0 and Theta_1 in the common Albany::ParamEnum... We could just use those, no?

Reply:
It seems that it is still needed; I changed all the rest based on your comments though.